src/vmm/src/vmm_config/machine_config.rs

```diff
@@ -252,6 +272,33 @@ pub enum CpuFeaturesTemplate {
     T2CL,
     /// T2A Template.
     T2A,
+
+    /// Custom cpu template
+    /// Not accessable to users
+    /// For internal use only
+    Custom,
+}
+
+/// Template types available for configuring the ARM CPU features that map
+/// to EC2 instances.
+#[cfg(target_arch = "aarch64")]
+#[derive(Clone, Copy, Debug, Deserialize, Eq, PartialEq, Serialize, Versionize)]
+pub enum CpuFeaturesTemplate {
```

Comment:
Given that developing hard-coded templates should become a legacy function, I'm not in favour of creating a new enum for one template, unless we plan to use this to accommodate more static templates in the future.